<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextWindowChangeValueDialog</class>
 <widget class="QDialog" name="TextWindowChangeValueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>371</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Change value</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,0,1">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="d_labelName">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="d_labelVariableName">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="d_groupBoxDisplaytype">
     <property name="title">
      <string>Displaytype</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="d_radioButtonDecimal">
          <property name="text">
           <string>decimal</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupDisplaytype</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="d_radioButtonBinary">
          <property name="text">
           <string>binary</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupDisplaytype</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="d_radioButtonHexadecimal">
          <property name="text">
           <string>hexadecimal</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupDisplaytype</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="d_radioButtonPhysically">
          <property name="text">
           <string>physically</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupDisplaytype</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="d_groupBoxIncDec">
     <property name="title">
      <string>Inc-/Decrement</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="d_labelStep">
          <property name="text">
           <string>Step</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PhysValueInput" name="d_lineEditStep" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="d_radioButtonLinear">
          <property name="text">
           <string>linear (x=x+step)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupLinearPerCent</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="d_radioButtonPerCent">
          <property name="text">
           <string>per cent (x=x+x*step)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupLinearPerCent</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Value</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="PhysValueInput" name="d_ValueInput" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhysValueInput</class>
   <extends>QWidget</extends>
   <header>PhysValueInput.h</header>
   <container>1</container>
   <slots>
    <signal>ValueChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TextWindowChangeValueDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TextWindowChangeValueDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>d_lineEditStep</sender>
   <signal>ValueChanged()</signal>
   <receiver>TextWindowChangeValueDialog</receiver>
   <slot>on_d_lineEditStep_ValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_d_lineEditStep_ValueChanged()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroupLinearPerCent"/>
  <buttongroup name="buttonGroupDisplaytype"/>
 </buttongroups>
</ui>
